src/main/java/seedu/mtracker/commons/Validate.java

```diff
@@ -47,6 +53,35 @@ public static void checkName(String name, String instrumentType) throws InvalidN
         }
     }
 
+    public static boolean isInvalidInstrument(String instrument) {
+        switch (instrument) {
+        case AddStockCommand.COMMAND_WORD:
+        case AddCryptoCommand.COMMAND_WORD:
+        case AddForexCommand.COMMAND_WORD:
+        case AddEtfCommand.COMMAND_WORD:
+            return false;
```

Comment:
Add break statement here. If its better to use an if else feel free to do so

Reply:
adding break here leads to an error of unreachable statement for some reason

Reply:
Ohh its cause u are returning, since that's the case I think its ok